packages/core/mesh/teleport-extension-gossip/src/gossip-extension.ts

```diff
@@ -78,7 +85,14 @@ export class GossipExtension implements TeleportExtension {
     }
     await this._opened.wait();
     invariant(this._rpc, 'RPC not initialized');
-    await this._rpc.rpc.GossipService.announce(message);
+    try {
+      await this._rpc.rpc.GossipService.announce(message);
+    } catch (err) {
+      // TODO(nf): always abort on RpcClosedError? gets stuck in retry loop otherwise
+      if (err instanceof RpcClosedError) {
```

Comment:
check if this is still needed after the change above

Reply:
didn't, so i reverted this bit